msibi/tutorials/propane/propane.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Propane Tutorial\n",
+    "\n",
+    "Created by Davy Yue 2017-06-14\n",
+    "\n",
+    "\n",
+    "### Import packages\n",
+    "\n",
+    "According to PEP8 standards, imports are grouped into \n",
+    "1. standard library imports \n",
+    "2. related third party imports \n",
+    "3. local application/library specific imports\n",
+    "\n",
+    "Group 1 includes `itertools`, `string`, and `os`.\n",
+    "\n",
+    "Group 2 includes `numpy`, which is stored in the keyword `np` and can be accessed with `np.[function]`.\n",
+    "\n",
+    "Group 3 includes `MSIBI`, `State`, `Pair`, and `mie` from the `msibi` package, which manages pair-potential derivation using MultiState Iterative Boltzmann Inversion (MS-IBI).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import itertools\n",
+    "import string\n",
+    "import os\n",
+    "\n",
+    "import numpy as np\n",
+    "\n",
+    "from msibi import MSIBI, State, Pair, mie"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Optional Code (might break program if uncommented and run)\n",
+    "\n",
+    "Code below modified from opty.py in the lj tutorial to accommodate files in the propane tutorial"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #\n",
+    "# # # # # # WARNING : ENTERING DANGEROUS TERRITORY # # # # # # \n",
+    "# # # # # # # # TRESPASSERS WILL BE SHOT ON SIGHT # # # # # # #\n",
+    "# # # # # # # # # SURVIVORS WILL BE SHOT AGAIN # # # # # # # #\n",
+    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #\n",
+    "#\n",
+    "# os.system('rm state*/* rdfs/C3* f_fits.log state*/log.txt')\n",
+    "# os.system('rm state*/err.txt')\n",
+    "# os.system('rm state*/query.dcd')\n",
```

Comment:
Let's make this a little more subtle; you can make a cell above the "dangerous" cell with a warning.